requirements.txt

```diff
@@ -1,9 +1,6 @@
-coverage
+coverage==4.0.3
 python-coveralls
-nose2
+nose2<=0.6.5
```

Comment:
Why do we need specific versions for these now?

Reply:
nose2 takes dependency on a specific version which conflicts with something else. It seems like the latest version of all of these things don't play well with each other.

Reply:
I see, did we decide that we have to use nose2 going forward?

Reply:
To clarify, the tests run twice on windows and fails on the second run through due to some environment issue. I did a preliminary search and didn't find a good fix.